src/xcode/ENA/ENA/Source/Scenes/Home/HomeTableViewController.swift

```diff
@@ -429,6 +430,13 @@ class HomeTableViewController: UITableViewController, NavigationBarOpacityDelega
 	}
 
 	private func showDeltaOnboardingIfNeeded(completion: @escaping () -> Void = {}) {
+		switch deltaOnboardingDone {
```

Comment:
I think this was done by the delta onboarding internally, is that not the case?

Reply:
No, I didn't find anything.